### PR TITLE
Top-level functions are shadowed by sub-module funcs

### PR DESCRIPTION
**Pull request by vvmnnnkv**
## Description
I've noticed that in torch -> tfjs conversion, `tf.reshape` is overridden by `tf.layers.reshape`, which has different arguments.
Correct translation would be `torch.reshape` -> `tf.reshape`, but `tf.layers.reshape` has same key in the funcs map and overwrites `tf.reshape` when the map is populated.
I'm trying to solve that by giving priority to functions that has shorter `attrs` path.

This kinda works, but it seems the problem needs more correct fix that would keep both funcs in the map but differentiate them by sub-module on mapping time?

## Affected Dependencies
n/a

## How has this been tested?
Unit tests

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

**Comment by Nolski** [comment]
This is related to #16

**Comment by vvmnnnkv** [comment]
Noticed issue in tfjs parsed .json file:
e.g. if you see tf.GraphModel in tfjs API, it's methods are listed w/o tf.GraphModel. prefix.
Doc parser doesn't add tf.GraphModel prefix, so all these methods appear as is they were in global scope.
I'm going to manually remove "add" from tfjs/1.5.1.json because it interferes with `tf.add()`, but will create separate issue to fix doc parser.

**Comment by vvmnnnkv** [comment]
Separate ticket: https://github.com/OpenMined/Threepio/issues/113

**Review by Nolski — APPROVED** [review]
